File: Advanced/PageAndRDL/CustomChart/C#/TestViewer/ViewerForm.cs
using System;
using System.Windows.Forms;

namespace TestViewer
{
	public partial class ViewerForm : Form
	{
		public ViewerForm()
		{
			InitializeComponent();
		}

		protected override void OnLoad(EventArgs e)
		{
			base.OnLoad(e);
			viewer1.LoadDocument(@"..\..\..\..\Report\Radar.rdlx");
		}
	}
}

File: Advanced/PageAndRDL/RtfControl/VB.NET/TestViewer/ViewerForm.vb
Imports System.IO
Imports GrapeCity.ActiveReports.Document

Public Class ViewerForm
	Inherits Form

	Public Sub New()
		InitializeComponent()
	End Sub

	Protected Overrides Sub OnLoad(ByVal e As EventArgs)
		MyBase.OnLoad(e)
		Dim reportFile = New FileInfo("..\..\..\..\Report\Rtf.rdlx")

		Dim report = New PageReport(reportFile)
		Dim document = New PageDocument(report)

		viewer1.LoadDocument(document)
	End Sub
End Class
